odxtools/write_pdx_file.py

```diff
@@ -119,6 +119,8 @@ def write_pdx_file(
         jinja_env.globals["make_bool_xml_attrib"] = make_bool_xml_attrib
 
         vars: Dict[str, Any] = {}
+        # set the index.xml short name tag to the filename without the file extension.
+        vars["ecu_name"] = os.path.splitext(os.path.basename(output_file_name))[0].replace(".", "_")
```

Comment:
those changes look unrelated to the rest of the PR, maybe move them to their own PR?

Reply:
also, this seems to be incorrect from a logical point of view: `output_file_name` is set by the `for` loop on line 100, so whatever it contains is just the first item of the last entry of `auxiliary_content_specifiers` (note that it also might not exist at all...)

Reply:
further, the `ecu_name` variable does not seem to be used by the writing code, so these two lines should probably be removed...